terraform/user.tf

```diff
@@ -0,0 +1,20 @@
+# Create the test user
+module "user" {
+  source = "github.com/cisagov/molecule-iam-user-tf-module?ref=bugfix%2Fadd-tags-for-role-assumption"
```

Comment:
Usual reminder about going back and changing this at some point after that branch gets merged...

Reply:
See commit d1af977.